app/bot.py

```diff
@@ -4,58 +4,162 @@
 from discord.ext import commands
 from dotenv import load_dotenv
 
-bot = commands.Bot(command_prefix='/')
 
-def main():
-    load_dotenv()
-    sys.stdout.flush()
-    bot.run(os.getenv("TOKEN"))
+reaction_emoji = '📌'
+
+prefix_map = {
+    reaction_emoji: ''
+}
+
+ingore_message_reactions_once = {
```

Comment:
This stuff could probably be better tracked with a class, but it works as is. I'd be happy yo refactor it at a later date when I get more free time if you'd like.